docs/UserGuide.md

```diff
@@ -3,21 +3,43 @@ layout: page
 title: User Guide
 ---
```

Comment:
I know these wasn't modified by your commit, but do you know what this table is for?

Reply:
Not sure, but I think it's being used for GitHub pages.
It corresponds to this:
![image](https://user-images.githubusercontent.com/70756909/160240402-38e5f810-e287-47f1-942e-1fb36f4da4c6.png)